app/assets/stylesheets/common.scss

```diff
@@ -581,6 +581,15 @@ header .search_forms,
   }
 }
 
+.search_forms {
+  .geolocate {
+    transition: filter 0.15s ease-in-out;
+  }
+  .btn:not(:hover) .geolocate {
+    filter: brightness(53%);
```

Comment:
I can see what this is doing, but I don't think it's the right approach. We should either:

* Add icons to the spritesheet, with the colours that we want (for light mode, and dark mode if necessary)
* Or we should take the approach of the "swap directions" button, which has inline SVG and automatic colouring.